src/shared/curate/parse.ts

```diff
@@ -76,7 +75,7 @@ export async function convertMeta(data: any, onError?: (error: string) => void):
   parser.prop('application path',     v => parsed.game.applicationPath     = str(v));
   parser.prop('curation notes',       v => parsed.game.curationNotes       = str(v));
   parser.prop('developer',            v => parsed.game.developer           = arrayStr(v));
-  parser.prop('extreme',              v => parsed.game.extreme             = str(v));
+  parser.prop('extreme',              v => parsed.game.extreme             = v === 'Yes' ? true : false);
```

Comment:
I don't think it's a good idea to go from parsing yes and true (both case insensitive) as true to only parsingYes (case sensitive) as true. This could break curations written or edited by hand.